packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/interact/content/useEnsAddress.ts

```diff
@@ -9,8 +9,8 @@ const mainnetClient = createPublicClient({
   transport: http(),
 });
 
-function isValidEnsName(name: string): boolean {
-  return name.includes(".");
+function isValidEnsName(name: string | undefined): name is string {
+  return !!name && name.includes(".");
 }
```

Comment:
erm does this mean `useEnsAddress` should also accept `undefined`?

seems like there's an inaccurate type floating around that we haven't fully plumbed through